primitives/benches/pcs.rs

```diff
@@ -39,7 +39,13 @@ pub fn bench_pcs_method<E: Pairing>(
             BenchmarkId::from_parameter(num_vars),
             &num_vars,
             |b, num_vars| {
-                b.iter(|| method(&pp, *num_vars));
+                b.iter_custom(|i| {
```

Comment:
What's the semantic difference here? And what is i? (Maybe add some comments for the reader to better understand the logic).

Reply:
I've added a comment about why we need `iter_custom`, trying to keep it brief. Some more info: https://bheisler.github.io/criterion.rs/book/user_guide/timing_loops.html.